pr_etiquette.md

```diff
@@ -0,0 +1,62 @@
+# Etiquette for pull requests
+
+## As an author (assuming non-committer)
```

Comment:
```suggestion
## As an author (developer w/o merge permissions)
```

Reply:
A non-committer might not be aware of the difference between committers and contributors.

Reply:
maybe it's better to leave out this qualification altogether. why would we have different rules for non-project members anyway.